### DISCUSSION TITLE
Webhooks affected when module is added

### DISCUSSION THREAD

**Discussion by Chippd** [Baroshem/nuxt-security#169] [Q&A]
Hi, 

I added the module but when I did, webhooks from two different services were erroring for me, as in, my application would not process the webhooks as it usually did. 

Is there an obvious reason for this that I'm missing? Other API calls seemed to work fine from within my app, but third parties not so much. 

Thanks!

**Comment by Baroshem** [comment]
Hi there,

Based on the explanation of the issues you are encountering, I would guess that it can be related to either ContentSecurityPolicy or CORS due to the relation with third party resources.

If you can create a stackblitz reproduction I could take a look at it and help you set the policies correctly :)
